app/views/error-pages/4xx.njk

```diff
@@ -8,6 +8,6 @@
     </div>
   </div>
   <div>
-    <a href="/">Go back home</a>
+    <p><a href="/" class="govuk-link">Go back home</a></p>
```

Comment:
The class doesn't need to be added because `application.scss` contains `$govuk-global-styles: true;` as per https://frontend.design-system.service.gov.uk/sass-api-reference/#global-styles :)